app/src/ingest_bem_pdfs.py

```diff
@@ -203,16 +203,28 @@ def _split_into_chunks(document: Document, grouped_texts: list[EnrichedText]) ->
         token_count = len(embedding_model.tokenizer.tokenize(paragraph.text))
         if token_count > embedding_model.max_seq_length:
             # Split the text into chunks of approximately equal length by characters,
-            # which doesn't necessarily mean equal number of tokens, but close enough
-            num_of_splits = math.ceil(token_count / embedding_model.max_seq_length)
+            # which doesn't necessarily mean equal number of tokens, but close enough.
+            # The arbitrary 1.5 tolerance factor tries to account for higher token counts per chunk when text is split.
+            num_of_splits = math.ceil((token_count * 1.5) / embedding_model.max_seq_length)
```

Comment:
I had to bump this up to 1.5 from 1.2 for tables in 541.pdf page 5. This is a quick fix for now, but we'll need to be smarter about grouping too much text together.